### DISCUSSION TITLE
Can't build setup file with latest version of WIX

### DISCUSSION THREAD

**Discussion by alexcardos** [wixtoolset/issues#6420] [Questions]
After updating from v3.11.1.2318 to v3.11.2.4516 , we can't build the setup file any more (Configuration: Release|x86). We have downgraded to the previous version and everything seems to be working fine again. Think there is a problem with the latest version of WIX.

**Comment by rseanhall** [comment]
The WiX Toolset does not use GitHub issues for support. I have converted this issue into a discussion item, which is an experiment that we're trying out. Alternatively, you may want to ask your question on the [wix-users mailing list](https://wixtoolset.org/documentation/mailinglist/) or [Stack Overflow](https://stackoverflow.com/questions/tagged/wix).

If there was a general problem then we would have certainly heard about it by now. Please provide more details.

**Comment by robmen** [comment]
There are many thousands of developers using WiX v3.11.2.4516 today. It is absolutely possible you hit an issue unique to your configuration or use case but we'd need a lot more information about what is not working to provide any amount of help.
